strawberryfields/apps/qchem/dynamics.py

```diff
@@ -64,7 +64,7 @@
 return a custom ``sf`` operation that contains the required unitary and rotation operations
 explained in steps 2-4 of the algorithm.
 """
-from typing import Any
+from types import FunctionType
```

Comment:
I usually do
```python
from typing import Callable
```
I think the `types` module's intended use is for within `isinstance()` checks : https://docs.python.org/3/library/types.html#standard-interpreter-types.

Reply:
Done. Thanks @trbromley